skymp5-front/src/interfaces/frame.ts

```diff
@@ -0,0 +1,5 @@
+import { DefaultButtonComponentProps } from './buttons';
+
+export interface SkyrimFrameProps extends DefaultButtonComponentProps {
+  header?: boolean
```

Comment:
`;`

Reply:
Since this is still draft PR, I haven't fixed all styling etc. Will fix when all intended functionality will be working :)

Reply:
Yep, I've just been told by @SupAidmi that the PR is working and they wanted to merge it, so decided to take a look :D